### PR TITLE
bump version to 1.18.1

### PR DESCRIPTION
**Pull request by mrocklin**
This we now depend on dask-core rather than dask

### CODE REVIEW

**Comment by jakirkham** [comment]
I think this needs a re-render. Also could you please disable CircleCI on your fork of the feedstock? Running CircleCI on the fork causes us problems when trying to test the merged result of PRs.

**Comment by mrocklin** [comment]
OK, I think I turned off Circle-CI.  I'm having some trouble with conda-smithy though

```
mrocklin@carbon:~/workspace/distributed-feedstock$ conda install conda-smithy -c conda-forge
Fetching package metadata ...........
Solving package specifications: .

Package plan for installation in environment /home/mrocklin/Software/anaconda:

The following packages will be SUPERSEDED by a higher-priority channel:

    conda: 4.3.25-py36_0 --> 4.3.23-py36_0 conda-forge

mrocklin@carbon:~/workspace/distributed-feedstock$ conda-smithy rerender
Traceback (most recent call last):
  File "/home/mrocklin/Software/anaconda/bin/conda-smithy", line 4, in <module>
    import conda_smithy.cli
  File "/home/mrocklin/Software/anaconda/lib/python3.6/site-packages/conda_smithy/cli.py", line 14, in <module>
    from . import ci_register
  File "/home/mrocklin/Software/anaconda/lib/python3.6/site-packages/conda_smithy/ci_register.py", line 10, in <module>
    from . import github
  File "/home/mrocklin/Software/anaconda/lib/python3.6/site-packages/conda_smithy/github.py", line 16, in <module>
    from . import configure_feedstock
  File "/home/mrocklin/Software/anaconda/lib/python3.6/site-packages/conda_smithy/configure_feedstock.py", line 48, in <module>
    from conda_build_all.version_matrix import special_case_version_matrix, filter_cases
  File "/home/mrocklin/Software/anaconda/lib/python3.6/site-packages/conda_build_all/version_matrix.py", line 19, in <module>
    for attr in ['Unsatisfiable', 'NoPackagesFound'])
  File "/home/mrocklin/Software/anaconda/lib/python3.6/site-packages/conda_build_all/version_matrix.py", line 19, in <genexpr>
    for attr in ['Unsatisfiable', 'NoPackagesFound'])
AttributeError: module 'conda.resolve' has no attribute 'NoPackagesFound'
```

**Comment by jakirkham** [comment]
`conda-build-all` requires `conda` 4.3.22 to work correctly, which affects `conda-smithy`. So would downgrade `conda` to 4.3.22 and retry.

xref: https://github.com/conda-tools/conda-build-all/issues/91

**Comment by mrocklin** [comment]
```
mrocklin@carbon:~/workspace/distributed-feedstock$ conda install conda-smithy conda=4.3.22 -c conda-forge
Fetching package metadata ...........
Solving package specifications: .

Package plan for installation in environment /home/mrocklin/Software/anaconda:

The following packages will be DOWNGRADED:

    conda: 4.3.23-py36_0 conda-forge --> 4.3.22-py36_0 conda-forge

mrocklin@carbon:~/workspace/distributed-feedstock$ conda-smithy rerender
No appveyor token. Create a token at https://ci.appveyor.com/api-token and
Put one in ~/.conda-smithy/appveyor.token
No anaconda token. Create a token via
  anaconda auth --create --name conda-smithy --scopes "repos conda api"
and put it in ~/.conda-smithy/anaconda.token
Fetching package metadata ...........
Traceback (most recent call last):
  File "/home/mrocklin/Software/anaconda/bin/conda-smithy", line 6, in <module>
    sys.exit(conda_smithy.cli.main())
  File "/home/mrocklin/Software/anaconda/lib/python3.6/site-packages/conda_smithy/cli.py", line 289, in main
    args.subcommand_func(args)
  File "/home/mrocklin/Software/anaconda/lib/python3.6/site-packages/conda_smithy/cli.py", line 194, in __call__
    configure_feedstock.main(args.feedstock_directory)
  File "/home/mrocklin/Software/anaconda/lib/python3.6/site-packages/conda_smithy/configure_feedstock.py", line 709, in main
    render_circle(env, config, forge_dir)
  File "/home/mrocklin/Software/anaconda/lib/python3.6/site-packages/conda_smithy/configure_feedstock.py", line 77, in render_circle
    forge_config.get('channels', {}).get('sources', tuple())
  File "/home/mrocklin/Software/anaconda/lib/python3.6/site-packages/conda_smithy/configure_feedstock.py", line 626, in compute_build_matrix
    mtx = special_case_version_matrix(meta, index)
  File "/home/mrocklin/Software/anaconda/lib/python3.6/site-packages/conda_build_all/version_matrix.py", line 223, in special_case_version_matrix
    py_vn = minor_vn(index[python_pkg.fn]['version'])
KeyError: 'python-3.3.0-pro0.tar.bz2'
```

**Comment by mrocklin** [comment]
I apologize for my ignorance in this space.  My ability to debug conda build issues is shamefully low.

**Comment by jakirkham** [comment]
Could you please include the results of `conda list -n root`?

**Comment by mrocklin** [comment]
```
mrocklin@carbon:~/workspace/distributed-feedstock$ conda list -n root
# packages in environment at /home/mrocklin/Software/anaconda:
#
_license                  1.1                      py36_1  
alabaster                 0.7.9                    py36_0  
altair                    1.2.0                    py36_0    conda-forge
anaconda                  custom                   py36_0  
anaconda-client           1.6.0                    py36_0  
anaconda-navigator        1.5.0                    py36_0  
anaconda-project          0.4.1                    py36_0  
args                      0.1.0                     <pip>
arrow-cpp                 0.3.pre             np112py36_0    conda-forge
astroid                   1.4.9                    py36_0  
astropy                   1.3                 np112py36_0    conda-forge
autoflake                 0.7                       <pip>
autopep8                  1.3.2                     <pip>
babel                     2.3.4                    py36_0  
backports                 1.0                      py36_0  
bcolz                     1.0.0                    py36_0  
beautifulsoup4            4.5.3                    py36_0  
bitarray                  0.8.1                    py36_0  
bkcharts                  0.2                      py36_0  
blas                      1.1                    openblas    conda-forge
blaze                     0.10.1                   py36_0  
blosc                     1.2.3                         0  
blosc                     1.5.1                     <pip>
bokeh                     0.12.4                    <pip>
bokeh                     0.12.3                    <pip>
bokeh                     0.12.7rc1                  py_0    bokeh/c/dev
boto                      2.45.0                   py36_0  
boto3                     1.4.2                    py36_0    conda-forge
botocore                  1.4.90                   py36_0  
bottleneck                1.2.1               np112py36_0  
bson                      0.4.6                     <pip>
BSON-NumPy                0.1                       <pip>
cached-property           1.3.0                     <pip>
cairo                     1.14.8                        0  
certifi                   2017.4.17                 <pip>
cffi                      1.9.1                    py36_0  
chardet                   3.0.4                     <pip>
chardet                   2.3.0                    py36_0  
chest                     0.2.3                    py36_0  
click                     6.7                      py36_0  
click-plugins             1.0.3                    py36_0    conda-forge
cligj                     0.4.0                    py36_0    conda-forge
clint                     0.5.1                     <pip>
cloudpickle               0.4.0                    py36_0  
clyent                    1.2.2                    py36_0  
colorama                  0.3.7                    py36_0  
colorama                  0.3.9                     <pip>
conda                     4.3.22                   py36_0    conda-forge
conda-build               2.1.7                    py36_0    conda-forge
conda-build-all           1.0.3                    py36_0    conda-forge
conda-env                 2.6.0                         0    conda-forge
conda-smithy              2.3.3                    py36_0    conda-forge
conda-verify              2.0.0                    py36_0    conda-forge
configobj                 5.0.6                    py36_0  
contextlib2               0.5.4                    py36_0  
conttest                  0.0.8                     <pip>
coverage                  4.3.4                     <pip>
cryptography              1.7.1                    py36_0  
curl                      7.52.1                        0  
cycler                    0.10.0                   py36_0  
cykdtree                  0.2.4                     <pip>
cython                    0.25.2                   py36_0  
cytoolz                   0.8.2                    py36_0    conda-forge
dask                      0.14.3                   py36_0  
dask-ec2                  0.4.0+16.g6f791f5           <pip>
dask-patternsearch        0.0.1                     <pip>
dask-searchcv             0.0.2                     <pip>
datashape                 0.5.4                    py36_0  
dbus                      1.10.10                       0  
decorator                 4.0.11                   py36_0  
descartes                 1.1.0                    py36_0    conda-forge
dill                      0.2.5                    py36_0  
distributed               1.16.3                   py36_0  
docker                    2.2.1                     <pip>
docker-compose            1.13.0                    <pip>
docker-pycreds            0.2.1                     <pip>
dockerpty                 0.4.1                     <pip>
docopt                    0.6.2                     <pip>
docutils                  0.13.1                   py36_0  
entrypoints               0.2.2                    py36_0  
et_xmlfile                1.0.1                    py36_0  
expat                     2.1.0                         0  
fastcache                 1.0.2                    py36_1  
filelock                  2.0.6                    py36_0    conda-forge
fiona                     1.7.6               np112py36_0    conda-forge
flake8                    3.3.0                    py36_0  
flask                     0.12                     py36_0  
flask-cors                3.0.2                    py36_0  
fontconfig                2.12.1                        2  
freetype                  2.5.5                         2  
freexl                    1.0.2                         1    conda-forge
funcsigs                  1.0.2                     <pip>
futures-compat            1.0                       py3_0  
gcsfs                     0.0.1                     <pip>
gdal                      2.1.3               np112py36_0    conda-forge
geopandas                 0.2.1                    py36_4    conda-forge
geos                      3.5.1                         1    conda-forge
get_terminal_size         1.0.0                    py36_0  
gevent                    1.2.1                    py36_0  
giflib                    5.1.4                         0    conda-forge
gitdb                     0.6.4                    py36_1    conda-forge
gitpython                 2.1.1                    py36_0    conda-forge
glib                      2.50.2                        1  
gprof2dot                 2016.10.13                <pip>
graphviz                  0.6                       <pip>
graphviz                  2.38.0                        5  
greenlet                  0.4.11                   py36_0  
gst-plugins-base          1.8.0                         0  
gstreamer                 1.8.0                         0  
h5netcdf                  0.3.1                    py36_0    conda-forge
h5py                      2.7.0               np112py36_0  
harfbuzz                  0.9.39                        2  
hdf4                      4.2.12                        0    conda-forge
hdf5                      1.8.17                        1  
heapdict                  1.0.0                    py36_1  
httplib2                  0.10.3                    <pip>
icu                       54.1                          0  
idna                      2.5                       <pip>
idna                      2.2                      py36_0  
imagesize                 0.7.1                    py36_0  
ipykernel                 4.5.2                    py36_0  
ipython                   5.1.0                    py36_0  
ipython_genutils          0.1.0                    py36_0  
ipywidgets                6.0.0                    py36_0    conda-forge
isort                     4.2.5                    py36_0  
itsdangerous              0.24                     py36_0  
jbig                      2.1                           0  
jdcal                     1.3                      py36_0  
jedi                      0.9.0                    py36_1  
jemalloc                  4.4.0                         0    conda-forge
jinja2                    2.9.4                    py36_0  
jmespath                  0.9.1                    py36_0    conda-forge
joblib                    0.9.4                    py36_0  
jpeg                      9b                            0  
json-c                    0.12                          0    conda-forge
jsonschema                2.5.1                    py36_0  
jsonschema                2.6.0                     <pip>
jupyter                   1.0.0                    py36_0    conda-forge
jupyter_client            4.4.0                    py36_0  
jupyter_console           5.0.0                    py36_0  
jupyter_core              4.2.1                    py36_0  
jupyterlab                0.26.4                   py36_0    conda-forge
jupyterlab_launcher       0.3.1                    py36_0    conda-forge
kazoo                     2.2.1                     <pip>
kealib                    1.4.7                         1    conda-forge
keras                     2.0.2                    py36_0  
Keras                     2.0.4                     <pip>
krb5                      1.14.2                        0    conda-forge
lazy-object-proxy         1.2.2                    py36_0  
libdap4                   3.18.3                        2    conda-forge
libffi                    3.2.1                         1  
libgcc                    5.2.0                         0  
libgfortran               3.0.0                         1  
libgpuarray               0.6.4                         0  
libiconv                  1.14                          0  
libnetcdf                 4.4.1.1                       4    conda-forge
libpng                    1.6.27                        0  
libpq                     9.5.4                         4    conda-forge
libsodium                 1.0.10                        0  
libspatialindex           1.8.5                         1    conda-forge
libspatialite             4.3.0a                       15    conda-forge
libtiff                   4.0.6                         3  
libtool                   2.4.2                         0  
libxcb                    1.12                          1  
libxml2                   2.9.4                         0  
libxslt                   1.1.29                        0  
line-profiler             2.0                       <pip>
llvmlite                  0.18.0                   py36_0  
locket                    0.2.0                    py36_1  
loky                      0.2.0                     <pip>
lxml                      3.7.2                    py36_0  
lz4                       0.10.1                    <pip>
lz4                       0.9.4                    py36_0  
mako                      1.0.6                    py36_0  
markupsafe                0.23                     py36_2  
matplotlib                2.0.0               np112py36_0  
mccabe                    0.6.1                    py36_0  
memory-profiler           0.47                      <pip>
mistune                   0.7.3                    py36_0  
mkl                       2017.0.1                      0  
mkl-service               1.1.2                    py36_3  
mock                      2.0.0                    py36_0  
mpi4py                    2.0.0                    py36_2  
mpich2                    1.4.1p1                       0  
mpmath                    0.19                     py36_1  
msgpack-python            0.4.8                    py36_0    conda-forge
multipledispatch          0.4.9                    py36_0  
munch                     2.1.1                    py36_0    conda-forge
nbconvert                 4.2.0                    py36_0  
nbformat                  4.2.0                    py36_0  
netcdf4                   1.2.7               np112py36_0    conda-forge
netifaces                 0.10.5                    <pip>
networkx                  1.11                     py36_0  
nltk                      3.2.2                    py36_0  
nose                      1.3.7                    py36_1  
notebook                  5.0.0                    py36_0    conda-forge
numba                     0.33.0              np112py36_0  
numexpr                   2.6.2               np112py36_0    conda-forge
numpy                     1.13.1                    <pip>
numpy                     1.12.0          py36_blas_openblas_200  [blas_openblas]  conda-forge
numpydoc                  0.6.0                    py36_0  
oauth2client              4.1.2                     <pip>
odo                       0.5.0                    py36_1  
openblas                  0.2.19                        1    conda-forge
openjpeg                  2.1.2                         2    conda-forge
openpyxl                  2.4.1                    py36_0  
openssl                   1.0.2k                        1  
pandas                    0.20.2              np112py36_0  
pandas-datareader         0.3.0.post0               <pip>
pango                     1.40.3                        1  
parquet-cpp               1.1.0pre                      0    conda-forge
partd                     0.3.7                    py36_0  
patchelf                  0.9                           1    conda-forge
path.py                   10.0                     py36_0  
pathlib2                  2.2.0                    py36_0  
patsy                     0.4.1                    py36_0  
pbr                       1.10.0                   py36_0  
pcre                      8.39                          1  
pdir                      0.2.2                     <pip>
pep8                      1.7.0                    py36_0  
pexpect                   4.2.1                    py36_0  
pickleshare               0.7.4                    py36_0  
pillow                    4.0.0                    py36_0  
pip                       9.0.1                    py36_1  
pixman                    0.34.0                        0  
pkginfo                   1.4.1                     <pip>
pkginfo                   1.2.1                    py36_0    conda-forge
pluggy                    0.4.0                     <pip>
ply                       3.9                      py36_0  
proj4                     4.9.3                         4    conda-forge
prompt_toolkit            1.0.9                    py36_0  
protobuf                  3.2.0                    py36_0    conda-forge
psutil                    5.2.1                    py36_0  
psutil                    5.2.2                     <pip>
psycopg2                  2.7.1                    py36_0    conda-forge
ptime                     0.0.1                     <pip>
ptyprocess                0.5.1                    py36_0  
py                        1.4.34                   py36_0  
py4j                      0.10.4                   py36_0    conda-forge
py4j                      0.10.4                    <pip>
pyasn1                    0.2.3                     <pip>
pyasn1                    0.1.9                    py36_0  
pyasn1-modules            0.0.9                     <pip>
pycodestyle               2.3.1                    py36_0  
pycosat                   0.6.1                    py36_1  
pycparser                 2.17                     py36_0  
pycrypto                  2.6.1                    py36_4  
pycurl                    7.43.0                   py36_2  
pyfftw                    0.10.4              np112py36_1    conda-forge
pyflakes                  1.5.0                    py36_0  
pygithub                  1.29                     py36_0    conda-forge
pygments                  2.1.3                    py36_0  
pygpu                     0.6.4                    py36_1  
pykafka                   2.5.0                     <pip>
pylint                    1.6.4                    py36_1  
pymongo                   3.4.0                     <pip>
pyopenssl                 16.2.0                   py36_0  
pyparsing                 2.1.4                    py36_0  
pyproj                    1.9.5.1                  py36_0    conda-forge
pyqt                      5.6.0                    py36_2  
pysal                     1.13.0                   py36_0    conda-forge
pyspark                   2.1.1                    py36_0    conda-forge
pytables                  3.3.0               np112py36_0    conda-forge
pytest                    3.1.3                    py36_0  
pytest-profiling          1.2.6                     <pip>
python                    3.6.0                         0  
python-dateutil           2.6.0                    py36_0  
python-dateutil           2.6.1                     <pip>
python-snappy             0.5                      py36_0    conda-forge
pytz                      2016.10                  py36_0  
pytz                      2017.2                    <pip>
pyyaml                    3.12                     py36_0  
pyzmq                     16.0.2                   py36_0  
qt                        5.6.2                         3  
qtawesome                 0.4.3                    py36_0  
qtconsole                 4.2.1                    py36_1  
qtpy                      1.2.1                    py36_0  
ray                       0.1.0                     <pip>
readline                  6.2                           2  
redis                     3.2.0                         0  
redis-py                  2.10.5                   py36_0  
requests                  2.12.4                   py36_0  
requests                  2.18.1                    <pip>
requests-file             1.4.1                     <pip>
requests-ftp              0.3.1                     <pip>
requests-toolbelt         0.7.1                     <pip>
rope                      0.9.4                    py36_1  
rsa                       3.4.2                     <pip>
rtree                     0.8.3                    py36_0    conda-forge
ruamel.yaml               0.13.14                  py36_0    conda-forge
ruamel_yaml               0.11.14                  py36_1  
Rx                        1.5.9                     <pip>
s3fs                      0.0.9                    py36_0    conda-forge
s3transfer                0.1.10                   py36_0  
scikit-image              0.12.3              np112py36_1    conda-forge
scikit-learn              0.18.2          np112py36_blas_openblas_200  [blas_openblas]  conda-forge
scipy                     0.19.0          np112py36_blas_openblas_200  [blas_openblas]  conda-forge
seaborn                   0.7.1                    py36_0  
setuptools                27.2.0                   py36_0  
shapely                   1.5.17              np112py36_3    conda-forge
simplegeneric             0.8.1                    py36_1  
singledispatch            3.4.0.3                  py36_0  
sip                       4.18                     py36_0  
six                       1.10.0                   py36_0  
smmap                     2.0.1                    py36_0    conda-forge
snakeviz                  0.4.1                     <pip>
snappy                    1.1.4                         1    conda-forge
snowballstemmer           1.2.1                    py36_0  
sockjs-tornado            1.0.3                    py36_0  
sortedcollections         0.5.3                    py36_0  
sortedcollections         0.5.3                     <pip>
sortedcontainers          1.5.7                     <pip>
sortedcontainers          1.5.3                    py36_0    conda-forge
sphinx                    1.5.4                    py36_0  
sphinx-rtd-theme          0.2.4                     <pip>
spyder                    3.1.2                    py36_0  
sqlalchemy                1.1.5                    py36_0  
sqlite                    3.13.0                        0  
statsmodels               0.8.0               np112py36_0    conda-forge
sympy                     1.0                      py36_0  
tabulate                  0.7.7                     <pip>
tblib                     1.3.0                    py36_0  
tblib                     1.3.2                     <pip>
tensorflow                1.1.0               np112py36_0  
terminado                 0.6                      py36_0  
texttable                 0.8.8                     <pip>
Theano                    0.9.0                     <pip>
theano                    0.9.0                    py36_0  
thriftpy                  0.3.9                    py36_0    conda-forge
tk                        8.5.18                        0  
toolz                     0.8.2                    py36_0    conda-forge
tornado                   4.5.1                    py36_0  
tox                       2.7.0                     <pip>
traitlets                 4.3.1                    py36_0  
twine                     1.8.1                     <pip>
ujson                     1.35                     py36_0  
unicodecsv                0.14.1                   py36_0  
urllib3                   1.21.1                    <pip>
util-linux                2.21                          0  
vega                      0.4.4                    py36_1    conda-forge
virtualenv                15.1.0                    <pip>
wcwidth                   0.1.7                    py36_0  
websocket-client          0.40.0                    <pip>
werkzeug                  0.11.15                  py36_0  
wheel                     0.29.0                   py36_0  
widgetsnbextension        2.0.0                    py36_0    conda-forge
wrapt                     1.10.8                   py36_0  
xarray                    0.9.6                    py36_0  
xerces-c                  3.1.4                         0  
xgboost                   0.6a2                     <pip>
xgboost                   0.6a2                    py36_0    conda-forge
xlrd                      1.0.0                    py36_0  
xlsxwriter                0.9.6                    py36_0  
xlwt                      1.2.0                    py36_0  
xz                        5.2.2                         1  
yaml                      0.1.6                         0  
zeromq                    4.1.5                         0  
zict                      0.1.2                    py36_0    conda-forge
zlib                      1.2.8                         3  
mrocklin@carbon:~/workspace/distributed-feedstock$ 
```

**Comment by jakirkham** [comment]
Please upgrade `conda-build` to 2.1.17 and `conda-build-all` to 1.1.0. Then please check if this works.

**Comment by mrocklin** [comment]
```
mrocklin@carbon:~/workspace/distributed-feedstock$ conda install conda-build=2.1.17 conda-build-all 1.1.0 conda-smithy conda=4.3.22 -c conda-forge
Fetching package metadata ...........

PackageNotFoundError: Package missing in current linux-64 channels: 
  - 1.1.0

mrocklin@carbon:~/workspace/distributed-feedstock$ conda install conda-build=2.1.17 conda-build-all 1.1 conda-smithy conda=4.3.22 -c conda-forge
Fetching package metadata ...........

PackageNotFoundError: Package missing in current linux-64 channels: 
  - 1.1

mrocklin@carbon:~/workspace/distributed-feedstock$ conda install conda-build=2.1.17 conda-smithy conda=4.3.22 -c conda-forge
Fetching package metadata ...........
Solving package specifications: .

Package plan for installation in environment /home/mrocklin/Software/anaconda:

The following packages will be UPDATED:

    conda-build: 2.1.7-py36_0 conda-forge --> 2.1.17-py36_0 conda-forge

conda-build-2. 100% |###################################################################| Time: 0:00:00   1.08 MB/s
mrocklin@carbon:~/workspace/distributed-feedstock$ conda-smithy rerender
No appveyor token. Create a token at https://ci.appveyor.com/api-token and
Put one in ~/.conda-smithy/appveyor.token
No anaconda token. Create a token via
  anaconda auth --create --name conda-smithy --scopes "repos conda api"
and put it in ~/.conda-smithy/anaconda.token
Fetching package metadata ...........
Traceback (most recent call last):
  File "/home/mrocklin/Software/anaconda/bin/conda-smithy", line 6, in <module>
    sys.exit(conda_smithy.cli.main())
  File "/home/mrocklin/Software/anaconda/lib/python3.6/site-packages/conda_smithy/cli.py", line 289, in main
    args.subcommand_func(args)
  File "/home/mrocklin/Software/anaconda/lib/python3.6/site-packages/conda_smithy/cli.py", line 194, in __call__
    configure_feedstock.main(args.feedstock_directory)
  File "/home/mrocklin/Software/anaconda/lib/python3.6/site-packages/conda_smithy/configure_feedstock.py", line 709, in main
    render_circle(env, config, forge_dir)
  File "/home/mrocklin/Software/anaconda/lib/python3.6/site-packages/conda_smithy/configure_feedstock.py", line 77, in render_circle
    forge_config.get('channels', {}).get('sources', tuple())
  File "/home/mrocklin/Software/anaconda/lib/python3.6/site-packages/conda_smithy/configure_feedstock.py", line 626, in compute_build_matrix
    mtx = special_case_version_matrix(meta, index)
  File "/home/mrocklin/Software/anaconda/lib/python3.6/site-packages/conda_build_all/version_matrix.py", line 223, in special_case_version_matrix
    py_vn = minor_vn(index[python_pkg.fn]['version'])
KeyError: 'python-3.3.0-pro0.tar.bz2'
```

**Comment by jakirkham** [comment]
Maybe try updating `conda-build-all` while pinning the rest?

**Comment by mrocklin** [comment]
OK, that seems to have worked.  I'm surprised that the `1.1.0` pin failed earlier.

```
mrocklin@carbon:~/workspace/distributed-feedstock$ conda install conda-build=2.1.17 conda-smithy conda=4.3.22 conda-build-all -c conda-forge
Fetching package metadata ...........
Solving package specifications: .

Package plan for installation in environment /home/mrocklin/Software/anaconda:

The following packages will be UPDATED:

    conda-build-all: 1.0.3-py36_0  conda-forge --> 1.1.0-py36_0  conda-forge

The following packages will be SUPERSEDED by a higher-priority channel:

    conda:           4.3.25-py36_0             --> 4.3.22-py36_0 conda-forge

conda-build-al 100% |###################################################################| Time: 0:00:00 964.51 kB/s
mrocklin@carbon:~/workspace/distributed-feedstock$ conda-smithy rerenderNo appveyor token. Create a token at https://ci.appveyor.com/api-token and
Put one in ~/.conda-smithy/appveyor.token
No anaconda token. Create a token via
  anaconda auth --create --name conda-smithy --scopes "repos conda api"
and put it in ~/.conda-smithy/anaconda.token
Fetching package metadata ...........
Fetching package metadata ...........
Fetching package metadata .............
Fetching package metadata .............

Re-rendered with conda-smithy 2.3.3.

You can commit the changes with:

    git commit -m "MNT: Re-rendered with conda-smithy 2.3.3"

These changes need to be pushed to github!

```

Thank you for your guidance with this process @jakirkham

**Comment by martindurant** [comment]
Does this repo need to be "readd"-ed for the new token, or is rerendering sufficient?

**Comment by jakirkham** [comment]
No problem. Glad it worked. I think there was an `=` missing previously, but not sure whether that would have caused the issue or not.

Thanks for re-rendering. Was a little concerned that CircleCI only built for Python 3.6 previously. Appears it is now running all Python builds, which is good.

**Comment by jakirkham** [comment]
> Does this repo need to be "readd"-ed for the new token, or is rerendering sufficient?

That shouldn't be necessary as I already readded `dask` and `distributed` recently. Also IIUC the token issue has been fixed for all other feedstocks as well.

**Comment by jakirkham** [comment]
Is `futures` suppose to be a Python 2 dependency? Seems there is an import issue in the Python 2.7 test.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -26,10 +26,9 @@ requirements:
 
   run:
     - python
-    - bokeh >=0.12.3
```
Comment (jakirkham) [inline recipe/meta.yaml]: Guessing we are just dropping this to avoid pull in lots of dependencies here, right?

Comment (mrocklin) [reply recipe/meta.yaml]: Yes, we'll put this into the dask metapackage instead

**Comment by mrocklin** [comment]
Tests pass.  Merging soon if there are no further comments.

**Comment by jakirkham** [comment]
Thanks @mrocklin.

**Pull request merged**